faq.md

```diff
@@ -141,3 +141,28 @@ to choose a source file whose compile command to use when opening a header.
 The heuristics are currently based on filesystem paths and can sometimes
 choose the wrong source file, though 
 [improvements](https://github.com/clangd/clangd/issues/123) are planned.
+
+## Why does clangd produce false or missing diagnostics?
+
+Most cases of false or missing diagnostics are configuration-related, i.e.
```

Comment:
i believe there's value in cutting to the chest and explaining details later on in FAQ entries. So I'd move this paragraph to bottom (or rather make the current second paragraph the first one).